macros/twitter_ads_extract_url_parameters.sql

```diff
@@ -0,0 +1,20 @@
+{% macro twitter_ads_extract_url_parameter(field, url_parameter) -%}
+
+{{ adapter.dispatch('twitter_ads_extract_url_parameter', 'twitter_ads_source') (field, url_parameter) }}
```

Comment:
based on the [dispatch](https://docs.getdbt.com/reference/dbt-jinja-functions/dispatch) docs, i wonder if we _need_ to wrap this in a `return()`

finally asked about `return` in [dbt slack](https://getdbt.slack.com/archives/C2JRRQDTL/p1702339437846329) so we'll see what folks say

Reply:
Yeah I am not sure, but then I guess the dispatch is not really needed at all for this since the intention is this macro will never be dispatched. I can confirm the code works as intended, but I never tested with the macro being wrapped in the return function. 

May be worthwhile to add it just to be consistent with the docs and how dbt writes their own macros.

Reply:
Updated code.